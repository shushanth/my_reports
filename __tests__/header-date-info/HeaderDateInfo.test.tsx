import HeaderDateInfo from "@/app/my-reports/headers-date-info/HeadersDateInfo";
import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";

describe("HeaderDateInfo", () => {
  it("renders component", async () => {
    render(<HeaderDateInfo />);
    const heading = screen.queryByTestId("HeaderDateInfo_page--component");
    expect(heading).toBeDefined();
  });

  it("renders component page unchanged", () => {
    const { container } = render(<HeaderDateInfo />);
    expect(container).toMatchSnapshot();
  });
});
