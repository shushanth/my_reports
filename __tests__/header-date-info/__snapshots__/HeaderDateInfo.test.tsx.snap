// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderDateInfo renders component page unchanged 1`] = `
<div>
  <div
    class="headers_date"
  >
    <div
      class="headers_date_headers"
    >
      <div
        class="heading"
      >
        <h1
          class="heading-level-page--h1"
        >
          My Reports
        </h1>
      </div>
      <div
        class="label"
      >
        <p
          class="label_level--primary label_size--small"
        >
          A detailed report of content removal activities, including infringements, media removals, and insights into impacted value and top offenders.
        </p>
      </div>
    </div>
    <div
      class="headers_date_date"
    >
      October 2024
    </div>
  </div>
</div>
`;
